.github/workflows/test.yml

```diff
@@ -39,6 +41,8 @@ jobs:
           python-version: '3.12'
       - name: install pytest and hypothesis
         run: pip install pytest hypothesis
+      - name: install libevent and pkg-config to find libevent
+        run: brew install pkg-config libevent
       - name: make dev
         run: make dev
       - name: make check (dev)
```

Comment:
make dev CFLAGS=-I$(brew --prefix libevent)/include

Reply:
Yep, also `LIBS=-L$(brew --prefix libevent)/lib` is needed. That's a strange behavior, because on my local MacOS machine it works without these flags.

Reply:
UPD: I tried this on a clean MacOS machine and it looks like I had some workarounds on my machine to make it work.